cmd/pocket_core/main.go

```diff
@@ -0,0 +1,19 @@
+// Pocket Core: This is the starting point of the CLI.
+package main
+
+import "fmt"
+
+const (
```

Comment:
Should we move this to a constants file?

Reply:
Yes, that was what I was thinking

Reply:
Accept this PR and I'll change this in the next one